Formula/rclone.rb

```diff
@@ -0,0 +1,131 @@
+require "language/go"
+
+class Rclone < Formula
+  desc "rsync for cloud storage"
+  homepage "http://rclone.org"
+  url "https://github.com/ncw/rclone/archive/v1.32.tar.gz"
+  sha256 "9b29478188f183a33252e00a2bb63eb21d6f9dc415fbdaa0865a31a524147466"
+  head "https://github.com/ncw/rclone.git"
```

Comment:
I think I agree that head is better to remove because now master branch doesn't pass the test and it shown by their repository: https://github.com/ncw/rclone/commits/master